code/extensions/GoogleSitemapExtension.php

```diff
@@ -33,7 +33,8 @@ public function canIncludeInGoogleSitemap()
             $can = $this->owner->getGooglePriority();
         }
 
-        $this->owner->invokeWithExtensions('alterCanIncludeInGoogleSitemap', $can);
+        // Allow override. In this case, since we don't know if this should be an "and" vs. "or" override, we'll use the LAST result (if any).
+        if ($override = $this->owner->invokeWithExtensions('alterCanIncludeInGoogleSitemap', $can)) $can = array_pop($override);
```

Comment:
The idea being if ONE plugin says to deny the permission then it should be denied. A `min` is just a hacky way of finding a falsey val.

I guess really it should be a loop over the results as `null` returns should be ignored.